zerver/lib/hotspots.py

```diff
@@ -33,6 +33,11 @@
         'description': _('Click here to start a new conversation. Pick a topic '
                          '(2-3 words is best), and give it a go!'),
     },
+    'intro_drafts': {
+        'title': _('Drafts'),
+        'description': _('Any unsent messages are automatically saved as drafts. '
+                         'Click on the drafts button to access them anytime.'),
```

Comment:
I would maybe change this to say "...access them at anytime."  Also, should we consider singular as opposed to plural?  So like "Any unsent message is automatically...".

Reply:
Sure, I can make the changes. At the time I just came up with a placeholder string for testing.